### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by alexAubin**
## Changes

- https://github.com/YunoHost-Apps/libreto_ynh/pull/12

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by alexAubin** [comment]
!testme

**Review by ericgaspar — APPROVED** [review]

**Comment by ericgaspar** [comment]
!testme

File: scripts/change_url [hunk 1]
```diff
@@ -17,7 +17,7 @@ old_domain=$YNH_APP_OLD_DOMAIN
 old_path=$YNH_APP_OLD_PATH
 
 new_domain=$YNH_APP_NEW_DOMAIN
-new_path=$YNH_APP_NEW_PATH
+new_path="/"
```
Comment (alexAubin) [inline scripts/change_url]: Uuuuuh not sure to understand this change, can't the app be installed on a subpath ?

**Review by alexAubin — COMMENTED** [review]
(LGTM otherwise)

File: scripts/change_url (hunk 1 above)
Comment (ericgaspar) [reply scripts/change_url]: oops, yes it does...

**Comment by alexAubin** [comment]
!testme

**Pull request merged**